### DISCUSSION TITLE
Sveltekit eslint  A11y: A form label must be associated with a control

### DISCUSSION THREAD

**Discussion by ctwhome** [saadeghi/daisyui#1043] [Q&A] [answered]
Labels are used in many places like modals, and dropdowns... but eslint keeps on warning:  `A11y: A form label must be associated with a control` 

I didn't find the way to disable this rule in .eslint.cjs: 
```js
"rules": {
    "a11y/label-has-associated-control": "off"
  }
```
Do you happen to know how to disable the warning?

**Comment by rhscjohn-dev** [comment, marked as answer]
For me, the easiest way is add the missing for="" to the label.  See 1st label in example.

The other way would be add `<!-- svelte-ignore a11y-label-has-associated-control -->` right above the label.  See 2nd label in code example.

code example
```
<div class="form-control w-full max-w-xs">
  <label for="" class="label">
    <span class="label-text">What is your name?</span>
    <span class="label-text-alt">Alt label</span>
  </label>
  <input type="text" placeholder="Type here" class="input input-bordered w-full max-w-xs" />
  <!-- svelte-ignore a11y-label-has-associated-control -->
  <label  class="label">
    <span class="label-text-alt">Alt label</span>
    <span class="label-text-alt">Alt label</span>
  </label>
</div>
```

You can copy the above code into a Svelte REPL and you should not see any warnings.

**Reply by ctwhome** [reply]
thanks I think the `for=""` will do it.

**Comment by ctwhome** [comment]
That can be done with for=" ", not really the ideal solution maybe.. But to
be honest what is ideal, right?

Thanks for the answer
